packages/doctype-caip10-link/src/caip10-link-doctype.ts

```diff
@@ -74,7 +73,8 @@ export class Caip10LinkDoctype extends Doctype {
         if (!linkedChainId) {
             throw new Error('Chain ID must be specified according to CAIP-10')
         }
-        return { header: DoctypeUtils.metadataToRecordHeader(metadata) }
```

Comment:
I'm pretty sure this is the last caller of `DoctypeUtils.metadataToRecordHeader`.  Can we delete it?